bills-app/src/main/webapp/script.js

```diff
@@ -26,3 +26,19 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addCode() {
+    getBillData();
+}
+
+function getBillData() {
+    //api key is declared in api.js as: 'key'
+    var url = 'https://api.propublica.org/congress/v1/bills/upcoming/house.json';
```

Comment:
```suggestion
    var url = "https://api.propublica.org/congress/v1/bills/upcoming/house.json";
```
nit, just to match double quote usage everywhere else in the file